apps/src/code-studio/pd/components/scholarshipDropdown.jsx

```diff
@@ -1,10 +1,11 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {FormGroup} from 'react-bootstrap'; // eslint-disable-line no-restricted-imports
+import DCDO from '@cdo/apps/dcdo';
 import Select from 'react-select';
 
 // update this to lock scholarships so that scholarship status can't be updated via the UI.
```

Comment:
Now that it's realiant on the DCDO flag should we remove this comment that encourages editing this line for updates?

Reply:
oh good thought!